Problem: AWS documentation defines event timestamp as ISO-8601

yrashk opened the pull request:
It specifies the following format: 1970-01-01T00:00:00.000Z

While the one Minio is using 20160814T124605Z

(See http://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html)

Solution: adhere to AWS documentation

harshavardhana commented on event-notifier.go:
```diff
@@ -60,7 +60,7 @@ func newNotificationEvent(event eventData) NotificationEvent {
 		EventSource:       "aws:s3",
 		AwsRegion:         region,
-		EventTime:         tnow.Format(iso8601Format),
+		EventTime:         tnow.Format("1970-01-01T00:00:00.000Z"),
```
@yrashk - you can use. `time.RFC3339`

harshavardhana replied:
> @yrashk - you can use. time.RFC3339

or @yrashk - `tnow.Format(timeFormatAMZ)`

yrashk committed:
Problem: AWS documentation defines event timestamp as 1970-01-01T00:00:00.000Z

While Minio is using 20160814T124605Z

(See http://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html)

Solution: adhere to AWS documentation
